Sprint-2/implement/querystring.js

```diff
@@ -4,13 +4,28 @@ function parseQueryString(queryString) {
     return queryParams;
   }
   const keyValuePairs = queryString.split("&");
+  console.log("key value pairs after spliting &:", keyValuePairs)
```

Comment:
It is never too early to develop the habit to keep the code clean by removing 
(or at least commenting out) unused code or debugging code.